README.md

```diff
@@ -236,17 +184,17 @@ npm run start:verify:webpack
 npm run start:verify:vite
 ```
 
-## VSCode integration
+### VSCode integration
 
 You can as well run [vscode tasks](./.vscode/launch.json) to start and debug the server in different modes and the client.
 
 ## Troubleshooting
 
 ### General
 
-If you use **monaco-languageclient** make sure you have a version of **monaco-editor** installed in your project that is compliant with **monaco-languageclient** and its peer dependency [@codingame/monaco-vscode-api](https://github.com/CodinGame/monaco-vscode-api).
+Again, if you use **monaco-languageclient** make sure you define matching [Overrides instructions](#overrides-instructions) in your local project to override any mismatching **monaco-editor** or **vscode** versions with you dependency tree.
 
-Ensure **monaco-editor** and **monaco-languageclient** are imported before you do any **monaco-editor** intialization. This ensures `monaco` and `vscode` (from **@codingame/monaco-vscode-api**) are imported beforehand. This is for example done like this in all examples contained in this repository.
+Ensure **monaco-editor**, **vscode** and **monaco-languageclient** are imported before you do any **monaco-editor** or vscode-api intialization or start using it.
```

Comment:
I'm dumb at JavaScript, can we give an example of the wrong and right way to import these items?

Reply:
Yes, adding an example code snippet here or referring to an existing example in the repo is a good idea.

Reply:
I will refer/link to our python language client example.